@here/harp-mapview/lib/MapView.ts

```diff
@@ -582,7 +582,7 @@ export interface MapViewOptions extends TextElementsRendererOptions {
 export const MapViewDefaults = {
     projection: mercatorProjection,
 
-    maxVisibleDataSourceTiles: 120,
+    maxVisibleDataSourceTiles: 300,
```

Comment:
If the view is tilted far enough more than 120 tiles become visible. Problem is that it is semi random which tiles get discarded for spherical projection. For planar projection the area of a tile on screen is used for ordering, but for spherical projection the area is not actually calculated but set to 1 if within view frustum. So sometimes tiles near to the camera get discarded instead of only discarding tiles that are further away.

Reply:
Ok, good to know, I would suggest to try with some semi-accurate approach, instead of calculating the area of tiles after view projection we may simply use their distance to camera inverted (instead of area in screen space), maybe that would help us - at least gives better results with small overhead.